src/py42/clients/orgs.py

```diff
@@ -63,25 +63,23 @@ def get_by_uid(self, org_uid, **kwargs):
         params = dict(idType=u"orgUid", **kwargs)
         return self._session.get(uri, params=params)
 
-    def get_by_name(self, org_name, **kwargs):
+    def get_all_by_name(self, org_name, **kwargs):
         """Gets the organization with the given name.
         `REST Documentation <https://console.us.code42.com/apidocviewer/#Org-get>`__
 
         Args:
             org_name (str): A name of an organization.
 
         Returns:
-            list: A list of :class:`py42.response.Py42Response` objects each containing an
-            organization that has the given name.
+            generator: An object that iterates over :class:`py42.response.Py42Response` objects
+            that each contain a page of organizations with the given name.
         """
         found_orgs = []
         response = self.get_all(**kwargs)
         for page in response:
-            org_list = page[u"orgs"]
-            for org in org_list:
-                if org[u"orgName"] == org_name:
-                    found_orgs.append(org)
-        return found_orgs
```

Comment:
Generator actually seems like way technically easier, but still not matter what we do here, it is weird.

For the generator method, we won't know the `totalCount` so we have to remove that from the response.
For the custom JSON method, we would have to somehow manually craft a `Py42Response` object or like modify the first page (`Py42Response`) from the `get_all()` call